WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardPromptsCardCell.swift

```diff
@@ -357,13 +357,11 @@ class DashboardPromptsCardCell: UICollectionViewCell, Reusable {
             return false
         }
 
-        let shouldDisplayCard = UserPersistentStoreFactory.instance().promptsEnabledSettings[siteID] ?? false
         guard let todaysPrompt = promptsService.localTodaysPrompt else {
-            // If there is no cached prompt, it can't have been skipped. So show the card.
-            return shouldDisplayCard
+            return false
```

Comment:
It should be `true`; good catch, thanks.

It was right in the comment: "If there is no cached prompt, it can't have been skipped. So show the card.". So no `localTodaysPrompt` means no skipped prompts, which means – show the card. I fixed it and returned the comment.

And the `shouldDisplayCard` check is no longer needed because it's not performed in a single place in `BlogDashboardService` for all cards.

Reply:
> And the `shouldDisplayCard` check is no longer needed because it's now performed in a single place in `BlogDashboardService` for all cards.

Makes sense and thanks for explaining this part as well 🙇